docs/build/_toc.json

```diff
@@ -8,6 +8,10 @@
     {
       "title": "Build circuits with Qiskit",
       "children": [
+        {
+          "title": "Classical feedforward and control flow",
+          "url": "/build/dynamic-circuits"
```

Comment:
Can this URL be made more specific? Right now it's highly generic. It's better to use a future-proof URL like `classical-feedforward-and-control-flow` or `dynamic-circuits-feedforward`.

You'll update here and the file name.

Reply:
done